torchinductor/graph.py

```diff
@@ -289,10 +289,16 @@ def output(self, target, args, kwargs):
             assert isinstance(value, TensorBox)
             value = value.data
             assert isinstance(value, ir.StorageBox)
+            value_storage_box = value
             value = value.data
             if not isinstance(value, InputBuffer) or value.get_name() != name:
                 # one of our inputs was mutated, need to turn that into a copy
                 ir.MutationLayout.realize_into(value, self.graph_inputs_original[name])
+                try:
+                    ind = self.graph_outputs.index(value_storage_box)
+                    self.graph_outputs[ind] = self.graph_inputs_original[name]
+                except Exception:
```

Comment:
What exception is this catching?  Perhaps this should be a more narrow catch.

Reply:
If input is not found in the outputs, `.index` would fail. I am catching a ValueError now.